main.js

```diff
@@ -1,13 +1,34 @@
+var hexData = [];
+var currentHexes;
+
+var newPaletteButtonSection = document.querySelector('.button-area');
+var newPaletteButton = document.querySelector('button');
+
+var mainColorBoxes = document.querySelectorAll('.color-container');
 var lockButton = document.querySelector('.main-display');
 
+window.addEventListener('load', getNewHexes);
+
 lockButton.addEventListener('click', function(event) {
     if (event.target.className === 'lock-box') {
         lockToggle(event.target);
     }
 });
 
-function getRandomHex() {
-    return (Math.floor(Math.random() * 16777216).toString(16).padStart(6, 0));
+newPaletteButtonSection.addEventListener('click', function(event) {
+    if(event.target.classList.contains('button-box-l') || event.target.id === 'new-palette' || event.target.classList.contains('button-box-r')) {
```

Comment:
We should consider refactoring line 19 in main.js condition in the future to include one condition for the 3 boxes since we'll have another similar box where we want to use the class for styling.